simplified-books-controller/src/main/java/org/nypl/simplified/books/controller/ProfileAccountCreateTask.kt

```diff
@@ -35,6 +38,7 @@ class ProfileAccountCreateTask(
       this.logger.debug("creating account for provider {}", this.accountProviderID)
       val accountProvider = this.resolveAccountProvider()
       val account = this.createAccount(accountProvider)
+      metrics?.logMetric(MetricEvent.LibraryAdded(account.id.uuid.toString()))
```

Comment:
Do we have a spec somewhere of what the various metric events are? I'm guessing what you probably want here is actually `account.provider.id...`. The provider ID is a publically known ID that tells us which account was created (NYPL, SimplyE Collection, etc). The `account.id` is a randomly generated unique ID and so likely has no meaning in terms of metrics.